src/components/views/settings/SetIdServer.js

```diff
@@ -127,28 +130,68 @@ export default class SetIdServer extends React.Component {
         return !!this.state.idServer && !this.state.busy;
     };
 
+    _continueTerms = (fullUrl) => {
```

Comment:
`_continueTerms` seems like a confusing name for something that saves the new server to storage... Maybe this should become:

```suggestion
    _saveIdServer = (fullUrl) => {
```

Reply:
indeed - will apply both suggestions manually.